.github/workflows/changelogs/changelogs.py

```diff
@@ -0,0 +1,639 @@
+
+            stripped_line = line.rstrip("\n")
+            if not stripped_line.strip():
+                issues.append(Issue(IssueType.EMPTY_LINE, file, line_no))
+                continue
+
+            if re.search(r"^- ", stripped_line):
+                # Start of a new entry
+                if entry_buf:
+                    # Wrap up the previous entry
+                    entry = self.get_entry_obj(entry_buf, file, line_no)
+                    issues.extend(self.validate_chlog_entry(entry))
+                    entries.append(entry)
+                    entry_buf = [stripped_line]
+                else:
+                    # First entry in the file
+                    entry_buf.append(stripped_line)
+            else:
+                # Successive lines of the entry
+                if entry_buf:
+                    if not re.search(r"^  [^ ]", stripped_line):
+                        # Successive lines must be indented by two spaces
+                        issues.append(Issue(IssueType.WRONG_INDENT, file, line_no))
+                    # Strip 1 whitespace from the left
+                    # 2 (indentation) - 1 (a literal space after the last line)
+                    entry_buf.append(stripped_line[1:])
+                elif re.search(r"^\s+- ", stripped_line):
+                    # No space allowed before the first entry line
+                    issues.append(Issue(IssueType.WRONG_INDENT, file, line_no))
+                else:
+                    # All entries must start with '- ' characters
+                    issues.append(Issue(IssueType.WRONG_START, file, line_no))
+
+            if len(stripped_line) > self.max_line_length:
+                issues.append(Issue(IssueType.LINE_TOO_LONG.format(self.max_line_length), file, line_no))
+            if re.search(self.regex.MULTIW, stripped_line):
+                issues.append(Issue(IssueType.MULTI_WHITESPACE, file, line_no))
+            if re.search(self.regex.TRAILINGW, stripped_line):
+                issues.append(Issue(IssueType.TRAIL_WHITESPACE, file, line_no))
+
+        # EOF
+        if entry_buf:
+            # Validate and append the last entry
+            entry = self.get_entry_obj(entry_buf, file, line_no + 1)
+            issues.extend(self.validate_chlog_entry(entry))
+            entries.append(entry)
+
+        return (issues, entries)
+
+    def validate_bsc(self, entry: type[Entry]) -> list[type[Issue]]:
+        """Validate Bugzilla trackers against a Bugzilla host"""
+
+        issues = []
+        # 'bnc' is the name of the tracker as defined in the trackers file
+        if 'bnc' in entry.trackers:
+            for tracker, bug_id in entry.trackers['bnc']:
+                try:
+                    bug = self.bzapi.getbug(bug_id)
+                    logging.debug(f"Bug #{bug_id} belongs to product '{bug.product}'")
+
+                    if not bug.product.startswith("SUSE Manager"):
```

Comment:
As far as I know security incidents use a different product.

Reply:
yes, for security the product is called
`SUSE Security Incidents`

Reply:
Also in some cases we have even different products. Some bugs we fix are reported for the SLES products, for example. Because of that, I made this one a non-severe issue. I guess it's fine if the test warns anything other than "SUSE Manager".

Reply:
it could also be that a bsc under `SUSE Security Incidents` could also  be restricted. I think we could start checking only for  SUMA and enhance this later